completion/zsh

```diff
@@ -4,6 +4,9 @@ _downgrade(){
   _arguments -S -C '*:packages:->pkg' \
              '--pacman[Specify pacman command]' \
              '--pacman-conf[Specify pacman configuration file]:config:_files' \
+             '--pacman-cache[Specify pacman cache location(s)]:config:_files' \
+             '--pacman-log[Specify pacman log-file location]:config:_files' \
+             '--maxdepth[Specify maximum depth for local cache find]' \
```

Comment:
Do you have to do something here to indicate it takes an argument?

Reply:
As far as I know, there is no autocomplete prompt for integers here. We can prompt it to complete files or paths as per the options above, but that is not the case for `--maxdepth`.

Reply:
Well, I wasn't expecting it to be smart about completing integers, I was just afraid that if you did

```
% downgrade --maxdepth <tab>
```

and it doesn't know that `--maxdepth` take _some_ argument, it'll start completing more `downgrade` options at that position, instead of stopping like actual `find` does:

![](https://files.pbrisbin.com/screenshots/screenshot.548719.png)

Reply:
Ah I understand now! That should not be hard to do. Let me look through something and update the PR.

Reply:
Fixed, now it will not complete any option for `--maxdepth` and `--pacman`. I modified such that it will complete directories and not files for `--pacman-cache`.